optuna/samplers/_tpe/sampler.py

```diff
@@ -147,10 +149,71 @@ def objective(trial):
                 Added in v2.2.0 as an experimental feature. The interface may change in newer
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.2.0.
+        group:
+            If this and ``multivariate`` are :obj:`True`, the multivariate TPE with the group
+            decomposed search space is used when suggesting parameters.
+            The sampling algorithm decomposes the search space based on past trials and samples
+            from the joint distribution in each decomposed subspace.
```

Comment:
How about adding the description for the mathematical property of decomposed subspaces? For example, "The decomposed subspaces are a partition of the parameter set. Each subspace is a maximal subset subject to any search spaces should be superset or disjoint set of the subspace."